### ISSUE TITLE
Can't fire hellbore cannons

### ISSUE THREAD

**Issue by tjareth** [JHGuitarFreak/UQM-MegaMod#31]
On Hard mode, equipped with Hellbore cannons in the first, second, third, and rear slot (full set), and also equipped with three shiva furnaces and two dynamos... and I am finding I cannot fire the guns--nothing appears to happen at all.   Am I doing something wrong?

**Comment by Serosis** [comment]
Let me look into it.  
I haven't seen this problem before.

If I can't replicate it I'll need to ask if you can upload your save here.

**Comment by tjareth** [comment]
I've been hunting high and low, and can't find where the save files are kept for windows 10, or anywhere that says the location.  If you can identify where to look I'll happy add it.

**Comment by tjareth** [comment]
I found it:  the location for Windows 10 is C:\Users\ (username)\AppData\Roaming\uqm-megamod\save

Please find the affected save file attached.  I am in battle with the Thraddash (which is usually when I debut my flagship for fighting) and firing doesn't seem to do anything.  "Active" flickers.

[uqmsave.00.zip](https://github.com/Serosis/UQM-MegaMod/files/4775261/uqmsave.00.zip)

**Comment by Serosis** [comment]
Well I tested your save out and it definitely can't fire the cannons.

I'm going to play around with it to see if I can find some answers as to why.

Also sorry for not giving a heads up as to the save location. Been quite busy lately.

**Comment by Serosis** [comment]
Figured it out almost immediately.

In Hard Mode when you stick a weapon module in a "splitter" node it doubles the amount of battery needed since that node fires two simultaneous shots.  
I didn't foresee that having all the slots filled up with Hellbores would effectively make it impossible to fire since the amount of battery needed would exceed the amount the ship has.

I'll add a Hard Mode warning about this in the readme.

It's a bit of a strategy to get a working offensive going so I crammed two Fusion Blasters in the splitter nodes and added 4000 R.U. to your save to get you up and running again.

[uqmsave.04.zip](https://github.com/Serosis/UQM-MegaMod/files/4777452/uqmsave.04.zip)
